### ISSUE TITLE
clearorphans command does not clear orphans on QT wallet display

### ISSUE THREAD

**Issue by mdpfeiffer** [PaycoinFoundation/paycoin#307]
clearorphans command does not clear orphans on QT wallet display in "transactions" tab.

**Comment by IngCr3at1on** [comment]
You're referring to the RPC command?

**Comment by mdpfeiffer** [comment]
Yep, RPC command.

**Comment by IngCr3at1on** [comment]
Yeah that's expected because QT already has a snazzy right click function that works on the transactions tab for clearing orphans from there.

Because it rebuilds that table on open, technically speaking it should remove them if you were to close and reopen the wallet after doing clearorphans in RPC.

Having said that, it may be possible to check if it's the QT wallet and call the QT function from the RPC function....
